src/lib/Scenes/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextArtist.tsx

```diff
@@ -0,0 +1,25 @@
+import { Join, Spacer } from "@artsy/palette"
+import { ArtworkContextArtist_artwork } from "__generated__/ArtworkContextArtist_artwork.graphql"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { ArtistArtworkGrid } from "./ArtworkGrids"
+
+export const ArtworkContextArtistFragmentContainer = createFragmentContainer<{
+  artwork: ArtworkContextArtist_artwork
+}>(
+  props => {
+    return (
+      <Join separator={<Spacer my={2} />}>
+        <ArtistArtworkGrid artwork={props.artwork} />
+      </Join>
```

Comment:
will the `Join` have more children soon? Because `Join` with one child is a no-op.

Reply:
Just talked to @sweir27 going to add two more grids.